### ISSUE TITLE
[question] How to upload a binary package that was just created in CI?

### ISSUE THREAD

**Issue by Knitschi** [conan-io/conan#10436]
Hi,

On my CI server I am creating a package by calling ```conan create```. After that I want to upload the binary package that was just created. The poblem I face now is that I do not know how I can automate uploading exactly the package that was created with the previous create call. I think I could use the full reference like ```conan upload MyLib/1.0.0@jenkins/stable:<binary package hash> -r myremote``` but I don't know how I can get the `<binary package hash>` part via the command line. How is this usually done?

Thanks for your time!

**Comment by Knitschi** [comment]
Sorry for bothering you. I think I found the solution on my own. I guess I will use something like

````
conan info MyLib/1.0.0@jenkins/stable --profile myCreateProfile --json myInfo.json
````

and then parse the created json file.

**Comment by memsharded** [comment]
Hi @Knitschi 

I think you can do it more directly with the ``conan create  --json=file.json``, that will also output a file that you can parse and get the package_id from there.

**Comment by memsharded** [comment]
good news. the ``package-list`` feature will be merged for 2.0.7 in https://github.com/conan-io/conan/pull/13928, that will implement this flow (and more):

```bash
conan create app --format=json > graph.json
conan list --graph=graph.json --format=json > pkglist.json
conan upload --list=pkglist.json -r=default
```
